### PR TITLE
change duration compute methods

### PR DESCRIPTION
**Pull request by ccmteorljh**
1,、修改了duration的计算方法，原来的是统计了最后一个pass的时间，但是计算examples_per_sec时im_num缺失所有pass的图片总和，现在修改duration为所有pass的时间总和；
2、skip_batch_num这个参数没有起到作用，现在修改了

### CODE REVIEW

File: fluid/resnet50.py [hunk 1]
```diff
@@ -195,9 +199,9 @@ def run_benchmark(model, args):
             im_num += label.shape[0]
 
     duration = time.time() - start_time
-    im_num = im_num - args.skip_batch_num * args.batch_size
+    #im_num = im_num - args.skip_batch_num * args.batch_size
```
Comment (ranqiu92) [inline fluid/resnet50.py]: please delete the codes commented

File: fluid/resnet50.py [hunk 2]
```diff
@@ -170,12 +170,16 @@ def run_benchmark(model, args):
...
         if iter == args.iterations:
-            break
+            iter = 0
         for batch_id, data in enumerate(train_reader()):
-            if iter == args.skip_batch_num:
+            if iter < args.skip_batch_num:
+                iter += 1
+                continue
+            if pass_id == 0:
                 start_time = time.time()
```
Comment (ranqiu92) [inline fluid/resnet50.py]: For all batches in pass 0, this line will be executed. Does this is what you want?

Comment (ccmteorljh) [reply fluid/resnet50.py]: modified

File: fluid/resnet50.py (hunk 1 above)
Comment (ccmteorljh) [reply fluid/resnet50.py]: done

File: fluid/resnet50.py [hunk 3]
```diff
@@ -170,12 +170,18 @@ def run_benchmark(model, args):
...
     iter = 0
     im_num = 0
+    start_time = time.time()
     for pass_id in range(args.pass_num):
+        every_pass_loss = []
+        every_pass_acc = []
         accuracy.reset(exe)
         if iter == args.iterations:
-            break
+            iter = 0
```
Comment (ranqiu92) [inline fluid/resnet50.py]: Is this modification right? What's the use of `args.iterations` ?

Comment (ccmteorljh) [reply fluid/resnet50.py]: I think it shoud be:
accuracy.reset(exe)
iter = 0
the args.iterations is used to  limit the iterations in each pass, so it used in:
for batch_id, data in enumerate(train_reader()):

File: fluid/resnet50.py [hunk 4]
```diff
@@ -198,7 +198,7 @@ def run_benchmark(model, args):
             buf_size=5120),
         batch_size=args.batch_size)
 
-    place = core.CPUPlace() if args.device == 'CPU' else core.CUDAPlace(0)
+    place = core.CPUPlace() if args.device == 'CPU' else core.GPUPlace(0)
```
Comment (ranqiu92) [inline fluid/resnet50.py]: I remember CUDAPlace is used in the latest PaddlePaddle. Could you refer to the source codes to confirm this.

File: fluid/resnet50.py [hunk 5]
```diff
@@ -209,15 +209,12 @@ def run_benchmark(model, args):
         label = np.array(map(lambda x: x[1], data)).astype('int64')
         label = label.reshape([-1, 1])
 
-    iter = 0
     im_num = 0
-    start_time = time.time()
     for pass_id in range(args.pass_num):
```
Comment (ranqiu92) [inline fluid/resnet50.py]: https://github.com/ccmteorljh/benchmark/blob/6e9b36b2cd318f308b80c984617f79f6eccdce5e/fluid/resnet50.py#L170    Maybe moving L170 before L213 is better.

File: fluid/resnet50.py (hunk 4 above)
Comment (ccmteorljh) [reply fluid/resnet50.py]: Done

File: fluid/resnet50.py (hunk 5 above)
Comment (ccmteorljh) [reply fluid/resnet50.py]: Done
